client/src/components/AdminCard/index.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Button, Pagination } from 'antd';
+import './style.css';
+
+const AdminCard = ({
+  imgUrl,
+  name,
+  description,
+  githbUrl,
+  student,
+  websiteLink,
+  studentNames,
+  ComProject,
+  CliProject,
+  editCard,
+  deleteCard,
+}) => {
+  return (
+    <>
+      <section className="card-container">
+        <ul>
+          <li>
+            <div className="name-img-card">
+              <div className="img-card">
+                <img src={imgUrl} alt="card" />
+              </div>
+              <div>
+                <h3>Name</h3>
+                <p>{name}</p>
+              </div>
+            </div>
+            {description && (
+              <div>
+                <h3>Description</h3>
+                <p>{description}</p>
+              </div>
+            )}
+            <div>
+              <h3>Github link</h3>
+              <NavLink exact to={githbUrl}>
```

Comment:
This link (githbUrl) will go to another web page, so you can't use NavLink or Link for this purpose.
it will go to ``` http://localhost:3000/admin/https://github.com/GSG-G8' ``` and this is not correct
``` <Link> ``` Provides declarative, accessible navigation around your application.
``` <NavLink> ``` A special version of the <Link> that will add styling attributes to the rendered element when it matches the current URL.
so the router's navigation methods are only for your app, not the whole web.
so you can use here ``` <a href> ```

Reply:
Yeah, I have corrected the errors here. Thank you 😊